draft-irtf-cfrg-vdaf.md

```diff
@@ -799,15 +799,18 @@ Some common functionalities:
 * `concat(parts: list[bytes]) -> bytes` returns the concatenation of the input
   byte strings, i.e., `parts[0] + ... + parts[len(parts)-1]`.
 
-* `from_be_bytes(encoded: bytes) -> int` computes the inverse of
-  `to_be_bytes()`.
+* `from_be_bytes(encoded: bytes) -> int` decodes a big-endian byte string,
+  i.e., returns the integer `x` for which `to_be_bytes(x, len(encoded)) ==
```

Comment:
```suggestion
  i.e., returns the integer `x` for which `to_be_bytes(x, len(encoded)) =
```

Reply:
Actually, I intended to use `==`, i.e., Python's notation for equality testing. I changed `from_le_bytes()` to be consistent with this.